src/rdpc/analysesProcessor.ts

```diff
@@ -83,7 +88,7 @@ export const analysisStream = async function* ({
   analysesFetcher: typeof fetchAnalyses;
   donorId?: string;
 }): AsyncGenerator<Analysis[]> {
-  const chunkSize = config?.chunkSize || 1000;
+  let chunkSize = config?.chunkSize || 100;
```

Comment:
There is a default set in the Config file, so we don't need the || 100 anymore, unless a value of 100 is special to this scenario.